ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -47,7 +47,7 @@
     )
     from databricks.sdk.service.sql import StatementState
 
-DATABRICKS_CONNECT_SUPPORTED_VERSION = "15.1.0"
+DATABRICKS_CONNECT_SUPPORTED_VERSION = "16.1.1"
```

Comment:
Is this required?

Reply:
The warning message or the pinning?

Reply:
The pinning, doesn't 15.1.0 work?

Reply:
Good point. I think there's just a compatibility issue with certain runtimes, but that won't affect much as those are pre-installed with the given runtime. I'll revert this for the recommendation so as to not confuse users :)